### PR TITLE
feat(stats): add coordinates of currently selected point (if any)

### PR DESCRIPTION
**Pull request by nicolas-goudry**

### CODE REVIEW

File: src/scene/selection.ts [hunk 1]
```diff
@@ -67,3 +68,23 @@ export const getTargetElements = (
...
     ? [appState.editingElement]
     : getSelectedElements(elements, appState);
+
+export const getSelectedPoint = (
+  elements: readonly NonDeletedExcalidrawElement[],
+  appState: AppState,
+) => {
+  if (
+    appState.editingLinearElement &&
+    appState.editingLinearElement.activePointIndex != null
```
Comment (lipis) [inline src/scene/selection.ts:78]:
```suggestion
    appState.editingLinearElement?.activePointIndex !== null
```

File: src/locales/en.json [hunk 2]
```diff
@@ -240,7 +240,8 @@
     "version": "Version",
     "versionCopy": "Click to copy",
     "versionNotAvailable": "Version not available",
-    "width": "Width"
+    "width": "Width",
+    "point": "Point"
```
Comment (lipis) [inline src/locales/en.json:244]: can you you put it after `height` in alphabetical order

File: src/scene/selection.ts (hunk 1 above)
Comment (nicolas-goudry) [reply src/scene/selection.ts:78]: Oh right, wasn't sure I could use optional chaining!

Comment (nicolas-goudry) [reply src/scene/selection.ts:78]: @lipis I used `!=` with the intent to check `activePointIndex` is not null nor undefined.

If you prefer, I can do:
```suggestion
    appState.editingLinearElement?.activePointIndex !== null &&
    appState.editingLinearElement?.activePointIndex !== undefined
```

**Review by lipis — APPROVED** [review]

**Comment by dwelle** [comment]
Not sure useful enough to warrant a new section in the stats dialog.

Btw, what's the use of displaying the point position in line-local coords?

**Comment by nicolas-goudry** [comment]
It's tied to selected element, would it then be more relevant to display it in element section?

Regarding use case, I needed this information while working on svg-to-excalidraw. It helped me understand where I was messing up with curves calculations. I thought it could be useful to have it anyway, so here's this PR.

**Comment by lipis** [comment]
Yes, I also think that it's useful when doing pixel perfect thingies.. Combined nicely with the grid.

**Pull request closed without merging**